### ISSUE TITLE
[Enh] Manage logs via logrotate

### ISSUE THREAD

**Issue by Jaxom99** [YunoHost-Apps/etherpad_mypads_ynh#108]
Hi, 

Thanks for the work so far ! The app works great, but I noticed that the logfile is not managed : `/var/log/etherpad_mypads/etherpad.log` is 72MB and growing... Maybe it will lead to troubles at some point.

I believe there are some ynh helpers to help logrotate manage it. Maybe it's a good "first issue" for new packagers ;-)

**Comment by Jaxom99** [comment]
Thanks for giving it a try ! It seems to be working, however I encountered a bug following the migration of the app (backup + import to a new instance). There is now an error sent to root@ :

```
/etc/cron.daily/logrotate:
error: skipping "/var/log/etherpad_mypads/etherpad.log" because parent directory has insecure permissions (It's world writable or writable by group which is not "root") Set "su" directive in config file to tell logrotate which user/group should be used for rotation.
error: skipping "/var/log/etherpad_mypads/installation.log" because parent directory has insecure permissions (It's world writable or writable by group which is not "root") Set "su" directive in config file to tell logrotate which user/group should be used for rotation.
run-parts: /etc/cron.daily/logrotate exited with return code 1
```
I remain available to help you debug this if needed :smiley_cat:

**Comment by ericgaspar** [comment]
Should be fixed with #114
